Automattic/wp-calypso
# Untangling: Add Advanced Tools -> Monitoring

Fixes:

- https://github.com/Automattic/dotcom-forge/issues/9611

## Proposed Changes

This PR adds the Advanced Tools -> Monitoring subtab under the `untangling/hosting-menu` flag.

<img width="1106" alt="image" src="https://github.com/user-attachments/assets/0a5f493b-9f83-4c88-a244-b7c0bb99c298">


## Why are these changes being made?

pbxlJb-6ye-p2

## Testing Instructions

1. Go to /sites
2. Click an Atomic site
3. Go to Advanced Tools -> Monitoring subtab.
4. Verify that this tool is working.
5. Regression testing:
    - Go to /sites?flags=-untangling/hosting-menu
    - Click an Atomic site
    - Verify that the original Monitoring tab is still working.

## Pre-merge Checklist
